orchestrator/orchestratorAutomation.ps1

```diff
@@ -26,6 +26,8 @@ Param (
    [string] $aifip,
    [Parameter(Mandatory = $false, ValueFromPipelineByPropertyName)]
    [string] $aifport
+   [Parameter(Mandatory = $false, ValueFromPipelineByPropertyName)]
+   [string] $cephport

```

Comment:
We should name it storage port as it could be anything tomorrow.

Reply:
Done